src/lib/footer/Footer.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+    import { getYear } from 'date-fns';
+    import Logo from '../logo/Logo.svelte';
+    import GitHubIcon from '../icons/GitHubIcon.svelte';
+    import PoweredByVercel from './PoweredByVercel.svelte';
+    import { VERSION } from 'svelte/compiler';
+</script>
+
+<footer>
+    <div class="container">
+        <div class="footer-row">
+            <ul class="horizontal-list horizontal-list--v-sm">
+                <li>
+                    <a href="https://github.com/svelteclub"
+                        ><GitHubIcon --width="1.3rem" --color="rgb(var(--grey-50))" --margin="0 1rem 0 0" /></a
+                    >
```

Comment:
```suggestion
                    <a href="https://github.com/svelteclub">
                        <GitHubIcon
                            --width="1.3rem"
                            --color="rgb(var(--grey-50))" 
                            --margin="0 1rem 0 0" />
                    </a>
```

Reply:
Had it like this before and prettier seems to have messed it up.

Reply:
Prettier can sometimes mess it up but it tries it's best, I fixed it for ya to help